### PR TITLE
Tweak atomic site account deletion error message to give more info

### PR DESCRIPTION
**Pull request by mokagio**
Jetpack iOS 21.1 has been rejected because the reviewer didn't find the account deletion error message they got on an account with Atomic sites satisfactory.

While we work on improving the flow, this copy tweak might be just enough to comply with Apple's guidelines.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Me/Account Settings/AccountSettingsViewController.swift
Comment (staskus) [inline WordPress/Classes/ViewRelated/Me/Account Settings/AccountSettingsViewController.swift]: The current copy might be sufficient. 

Although, we could consider using even more precise language that [Apple is using](loper.apple.com/support/offering-account-deletion-in-your-app), to remove any ambiguity about the possibility of closing the account.

```suggestion
            value: "This user account cannot be closed immediately because it has active purchases. Please contact our support team to finish deleting the account.",
```

**Review by staskus — APPROVED** [review]
Since we want to be quick, my comment is a non-blocker. I still see the new message as an improvement to the previous one.

File: WordPress/Classes/ViewRelated/Me/Account Settings/AccountSettingsViewController.swift
Comment (mokagio) [reply WordPress/Classes/ViewRelated/Me/Account Settings/AccountSettingsViewController.swift]: Neat! Applied. Thanks.

**Comment by mokagio** [comment]
Thank you for the review and suggestion @staskus 🙌 

Once CI is green, I'll merge this and move on with merging the code freeze into `trunk` so that our bots can pick up the new localizations and send them to GlotPress.

Still TBD whether to cherry pick this change into an hotfix or not, but that would be a dedicated PR anyway.

**Comment by osullivanchris** [comment]
Thanks for pinging me for review. Apologies I had 1,400 unread in my inbox after leave, and only got to it now. Let's see if they approve this one. If not, let me know if you want me to flesh out the details on the alternate proposal I had.

Feel free to ping me on any further PRs, and slack me if you're ever waiting on a response. Thanks!

**Comment by mokagio** [comment]
@osullivanchris no worries at all 😄

**Pull request merged**
